e2e/s3/swupdate/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:latest AS build
+FROM ubuntu:plucky AS build
```

Comment:
This is a non-LTS release, I think noble would be better as it's the latest LTS and should have a longer support period.